code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -54,7 +54,7 @@
 	. = ..()
 	receiver.cure_blind(NO_EYES)
 	apply_damaged_eye_effects()
-	refresh(receiver, call_update = TRUE)
+	refresh(receiver, call_update = !special)
```

Comment:
Eyes would ALWAYS force a body update, even during special because someone forget to set this flag